tests/vmi_cloudinit_test.go

```diff
@@ -65,12 +67,16 @@ var _ = Describe("[rfe_id:151][crit:high][vendor:cnv-qe@redhat.com][level:compon
 		MountCloudInitConfigDrive func(*v1.VirtualMachineInstance)
 		CheckCloudInitFile        func(*v1.VirtualMachineInstance, string, string)
 		CheckCloudInitMetaData    func(*v1.VirtualMachineInstance, string, string)
+		CheckCloudInitIsoSize     func(vmi *v1.VirtualMachineInstance, source cloudinit.DataSourceType)
 	)
 
 	tests.BeforeAll(func() {
 		virtClient, err = kubecli.GetKubevirtClient()
 		tests.PanicOnError(err)
 
+		// from default virt-launcher flag: do we need to make this configurable in some cases?
```

Comment:
Not that I know of, but we should get rid of this global `SetLocalDirectory` at some point. In general difficult for testing.